From 59ee7ea5356072c1a601b2ed170b61ce3e72b0d3 Mon Sep 17 00:00:00 2001
From: Joe Perches <joe@perches.com>
Date: Mon, 12 Dec 2016 14:26:55 -0800
Subject: [PATCH 2977/5855] treewide: Make remaining source files
 non-executable

.c and .h source files should not be executable, change
the permissions to 0644.

[ This would normally go through Andrew Morton, but his ancient
  patch-based toolchain doesn't do permission changes ]

Signed-off-by: Joe Perches <joe@perches.com>
Acked-by: David Howells <dhowells@redhat.com>
Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
---
 drivers/gpu/drm/amd/include/asic_reg/dce/dce_11_2_d.h       | 0
 drivers/gpu/drm/amd/include/asic_reg/dce/dce_11_2_sh_mask.h | 0
 drivers/gpu/drm/amd/include/cgs_common.h                    | 0
 drivers/gpu/drm/amd/powerplay/smumgr/fiji_smumgr.c          | 0
 4 files changed, 0 insertions(+), 0 deletions(-)
 mode change 100755 => 100644 drivers/gpu/drm/amd/include/asic_reg/dce/dce_11_2_d.h
 mode change 100755 => 100644 drivers/gpu/drm/amd/include/asic_reg/dce/dce_11_2_sh_mask.h
 mode change 100755 => 100644 drivers/gpu/drm/amd/include/cgs_common.h
 mode change 100755 => 100644 drivers/gpu/drm/amd/powerplay/smumgr/fiji_smumgr.c

diff --git a/drivers/gpu/drm/amd/include/asic_reg/dce/dce_11_2_d.h b/drivers/gpu/drm/amd/include/asic_reg/dce/dce_11_2_d.h
old mode 100755
new mode 100644
diff --git a/drivers/gpu/drm/amd/include/asic_reg/dce/dce_11_2_sh_mask.h b/drivers/gpu/drm/amd/include/asic_reg/dce/dce_11_2_sh_mask.h
old mode 100755
new mode 100644
diff --git a/drivers/gpu/drm/amd/include/cgs_common.h b/drivers/gpu/drm/amd/include/cgs_common.h
old mode 100755
new mode 100644
diff --git a/drivers/gpu/drm/amd/powerplay/smumgr/fiji_smumgr.c b/drivers/gpu/drm/amd/powerplay/smumgr/fiji_smumgr.c
old mode 100755
new mode 100644
-- 
2.7.4

